homeassistant/components/smartthings/climate.py

```diff
@@ -529,13 +529,12 @@ def swing_mode(self) -> str:
 
     def _determine_preset_modes(self) -> list[str]:
         """Return a list of available preset modes."""
-        modes = []
         supported_modes = self._device.status.attributes[
             "supportedAcOptionalMode"
         ].value
         if WINDFREE in supported_modes:
-            modes.append(WINDFREE)
-        return modes
+            return [WINDFREE]
+        return []
```

Comment:
This should return `None` and not an empty list if there is no presets

Reply:
ok, thanks